customStep.js

```diff
@@ -0,0 +1,8 @@
+
+// in this file you can append custom step methods to 'I' object
+
+const customActor = () => {
+  const I = actor();
+  return I;
+};
+module.exports = customActor();
```

Comment:
leave an empty line after the last statement to all statements wherever this warning small symbol displays